roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
> dns can serve everything ...
Let's do that.  

This only effects the dhcp server. it separates the dns config from the servers etc/hosts, andI am ok with that.   

replacing "127.0.1.1 dc10b" is too hard for me, and I don't see much benefit.